### PR TITLE
[WIP] Claims Verification SIP

### PR DESCRIPTION
**Pull request by mistermoe**
Still a WIP. creating a Draft PR for visibility

_Note: SIP # is a placeholder incase other SIPs are submitted and pulled in before this one_

### CODE REVIEW

**Comment by mistermoe** [comment]
swap excalidraw illustrations for mermaid sequence diagrams

File: sip/sips/sip20220926/README.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+# Specification
+
+![application-submission](https://i.imgur.com/NgSVx53.png)
+
+![credential issuance](https://i.imgur.com/mHn8qa8.png)
+
+
+
+The overall desired mechanism can be broken down into the following questions:
+* How does is an Application Submitted?
```
Comment (nitro-neal) [inline sip/sips/sip20220926/README.md:43]: fix wording

**Review by decentralgabe — COMMENTED** [review]
I'd like to consider this in a broader frame, not specific to credential applications

**Comment by mistermoe** [comment]
image
 yup sg. interested to hear what you've got in mind. no rush. there's a few easy ways to generalize this.

**Pull request closed without merging**